sw/device/lib/dif/dif_kmac.h

```diff
@@ -150,7 +150,9 @@ typedef struct dif_kmac_config {
 
   /**
    * Entropy fast process mode when enabled prevents the KMAC unit consuming
-   * entropy unless it is processing a secret key.
+   * entropy unless it is processing a secret key. This process should not be
+   * used when resistance against side-channel attacks is requiered, because
```

Comment:
```suggestion
   * used when resistance against side-channel attacks is required, because
```

Reply:
Thanks for catching this. It is corrected now.